void_migration/void_migration.py

```diff
@@ -546,6 +765,9 @@ def time_march(p):
     plotter.plot_s_bar(y, s_bar_time, nu_time, p)
     plotter.plot_u_time(y, u_time, nu_time, p)
     np.save(p.folderName + "nu_t_x.npy", nu_time_x)
+    np.save(p.folderName + "mass_in_fun.npy", mass_in_fun)
+    np.save(p.folderName + "mass_count.npy", p_count)
+    np.save(p.folderName + "nu_non_zero_avg.npy", non_zero_nu_time)
```

Comment:
the way saving is controlled has been updated in the new version. can you move this across?